scipy/sparse/_coo.py

```diff
@@ -699,6 +699,7 @@ class coo_array(_coo_base, sparray):
 
     Sparse arrays can be used in arithmetic operations: they support
     addition, subtraction, multiplication, division, and matrix power.
+    Sparse arrays in COOrdinate format do not support slicing.
```

Comment:
The COO doc_string already lists a disadvantage being that it does not support slicing. Maybe we don't need anything further?